images/runtime/node/installDependencies.sh

```diff
@@ -30,7 +30,7 @@ echo "Version of npm: $currentNpmVersion"
 # Upgrade npm to the latest available version
 if [[ $nodeVersionMajor -ge 10  ]]; then
     echo "Upgrading npm..."
-    npm install npm@6.14.10 -g -loglevel silent
+    npm install npm@$NPM_VERSION -g -loglevel silent
```

Comment:
If node version is greater than 10, they'll all have the same npm version in the image? 
What if we want to keep some latest node version have npm 7 instead?

Reply:
we dont have any npm 7 requirement yet ..  when we start supporting npm 7 in a feature some of these  codes needs to be revisited for sure. This is to make sure we have coverage for what we have now